magefile.go

```diff
@@ -340,6 +382,41 @@ func pushImagesTo(registry string, info releases.GitMetadata) {
 	}
 }
 
+func PublishServerMultiArchImages() {
```

Comment:
This seems to be separate from the key focus on this PR maybe?
If so, I'm fine with it staying in here, but maybe a good call out in the PR desc

Reply:
This publishes an image for use with the operator. I'll note this in the PR description.